dev-python/tqdm/tqdm-4.65.0.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="Add a progress meter to your loops in a second"
+DESCRIPTION="tqdm derives from the Arabic word *taqaddum* (تقدّم) which can mean *progress*, and \
+is an abbreviation for *I love you so much* in Spanish *te quiero demasiado*.
+Instantly make your loops show a smart progress meter - just wrap any iterable with \
+``tqdm(iterable)``, and you're done!"
+HOMEPAGE="https://github.com/tqdm/tqdm"
+COPYRIGHT="2015-2023 tqdm contributors"
+LICENSE="MIT
+	MPL v2.0"
+REVISION="1"
+pypiVersion="3d/78/81191f56abb7d3d56963337dbdff6aa4f55805c8afd8bad64b0a34199e9b"
+SOURCE_URI="https://files.pythonhosted.org/packages/$pypiVersion/$portName-$portVersion.tar.gz"
```

Comment:
Doesn't work with https://github.com/tqdm/tqdm/archive/refs/tags/v4.65.0.tar.gz ?

Reply:
You should take care with using `$portNam`e also, if this wasn't `any` it would become `tqdm_x86` then (32bit secondary arch)

Reply:
> Yup, noticed that on other recipes. Would be better to just not use `$portName` there ever, just in case?

IMHO nothing architecture should be used in SOURCE_URI (my 2 cents)